### DISCUSSION TITLE
clarification on kerbs where this way meets this road

### DISCUSSION THREAD

**Discussion by IAmTheWebb** [streetcomplete/StreetComplete#4005] [Q&A] [answered]
Hi,
For the 'Are there kerbs where this way meets this road here? What kind?' quest, if the way continues at the other side of the road but has a different level of kerb I don't know which to enter.  

![Imagepipe_0](https://user-images.githubusercontent.com/17071553/166410886-898c9f14-beda-4e4e-b966-5eec1b498025.jpg)

**Comment by Helium314** [comment, marked as answer]
See #3246: best to leave a note
